### DISCUSSION TITLE
Microsoft 365 domains (DNS)

### DISCUSSION THREAD

**Discussion by plettb** [polymind-inc/acmebot#463] [Ideas]
I just started a different discussion about this topic (see here: https://github.com/shibayan/keyvault-acmebot/discussions/462).  I've since done a little research and found that it should be possible via Microsoft Graph (see https://docs.microsoft.com/en-us/graph/api/resources/domain?view=graph-rest-1.0).  I may even start on it, but am unlikely to have enough time to do it justice.

**Comment by shibayan** [comment]
As noted in the comments below, it is best to use Azure DNS; it is unlikely from a maintenance perspective that a Microsoft 365 DNS-enabled provider will ever be implemented.
https://github.com/shibayan/keyvault-acmebot/discussions/462#discussioncomment-2445676

**Reply by plettb** [reply]
Suggesting a different product does not actually answer the question.
